validators/types_unix.go

```diff
@@ -28,7 +28,12 @@ import (
 var DefaultSysSpec = SysSpec{
 	OS: "Linux",
 	KernelSpec: KernelSpec{
-		Versions: []string{`^3\.[1-9][0-9].*$`, `^([4-9]|[1-9][0-9]+)\.([0-9]+)\.([0-9]+).*$`}, // Requires 3.10+, or newer
+		// runc cgroup v2 https://github.com/opencontainers/runc/blob/3778ae603c706494fd1e2c2faf83b406e38d687d/docs/cgroup-v2.md?plain=1#L24
+		Versions: []string{`^3\.[1-9][0-9].*$`, `^([4-9]|[1-9][0-9]+)\.([0-9]+)\.([0-9]+).*$`}, // Requires 4.15+, or newer
+		// cgroup v2: Kubernetes recommended kernel version
+		// https://kubernetes.io/docs/concepts/architecture/cgroups/
+		RecommendedVersions: []string{`^5\.[8-9].*$`, `^5\.[1-9][0-9].*$`, `^([6-9]|[1-9][0-9]+)\.([0-9]+)\.([0-9]+).*$`}, // Requires 5.8+, or newer for cgroup v2
+		RecommendedNote:     "kernel version should >= '4.15', and 5.8+ is recommended.",
```

Comment:
can be called `VersionsNote`

from a discussion on slack we had:
> the checks are regexp based so could be 4.4 == ok, 4.19 == ok, any other >= 4.19 version == ok, any other version == err.

so  in this note we can say that all those 4.x LTS versions are supported but the recommendation is to use 4.15 or 5.8+.

Reply:
```
VersionsNote: "kernel version should be in maintenance, at least 4.4 or 4.19+, and to use cgroup v2, 5.8+ is recommended.",
```

update it as above..